CONTRIBUTING.md

```diff
@@ -180,6 +181,19 @@ base](https://github.com/mlflow/mlflow/blob/cdc6a651d5af0f29bd448d2c87a198cf5d32
 For more information about Plugins, see
 <https://mlflow.org/docs/latest/plugins.html>.
 
+### Python Style Guide
+##### Docstrings
+We follow [Google's Python Style Guide](https://google.github.io/styleguide/pyguide.html) for writing 
+docstrings. Make sure your docstrings adhere to this style guide.
+
+Currently, the MLflow team is working to reformat docstrings to adhere to 
+these guidelines. Any contributions regarding style reformatting are much appreciated!
+
+###### Code Style
+For Python code, we generally adhere to Google's Python Style Guide. However, we use [pylint](https://pypi.org/project/pylint/) 
+and [black](https://black.readthedocs.io/en/stable/the_black_code_style/index.html) in our CI via pre-commit Git hooks.
+If your code passes the CI checks, it's formatted correctly.
```

Comment:
I think we should mention ruff here: https://github.com/astral-sh/ruff

Reply:
Can we remove `we generally adhere to Google's Python Style Guide` because we don't?